src/MonitoringSolutions/docs/Get-AzMonitorLogAnalyticsSolution.md

```diff
@@ -41,23 +41,50 @@ Retrieves the user solution.
 
 ## EXAMPLES
 
-### Example 1: {{ Add title here }}
+### Example 1: Get a monitor log analytics solution by name
 ```powershell
-PS C:\> {{ Add code here }}
+PS C:\> Get-AzMonitorLogAnalyticsSolution -ResourceGroupName azureps-monitor -Name 'Containers(azureps-monitor)'
 
-{{ Add output here }}
+Name                      Type                                     Location
+----                      ----                                     --------
+Containers(azureps-monitor) Microsoft.OperationsManagement/solutions West US 2
 ```
 
-{{ Add description here }}
+This command gets a monitor log analytics solution by name.
 
-### Example 2: {{ Add title here }}
+### Example 2: Get a monitor log analytics solution by object
 ```powershell
-PS C:\> {{ Add code here }}
+PS C:\> $monitor = Get-AzMonitorLogAnalyticsSolution -ResourceGroupName azureps-monitor -Name 'Containers(azureps-monitor)'
```

Comment:
I think the correct way to call `GetViaIdentity` is to:

1. constrcut a hashtable containing the identity information of the resource
1. pass the hashtable to `GetViaIdentity` to get the full resource

For example:

```powershell
@{Id = "/subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f/resourcegroups/yemingmonitor/providers/microsoft.operationalinsights/workspaces/yemingmonitor"} | Get-AzMonitorLogAnalyticsSolution
```

Reply:
Add a new example description for this.